yellowbrick/classifier/rocauc.py

```diff
@@ -219,7 +247,16 @@ def draw(self):
         colors = self.colors[0:len(self.classes_)]
         n_classes = len(colors)
 
-        # Plot the ROC curves for each class
+        # If it's a binary decision, plot the single ROC curve
+        if self._binary_decision == True:
+            self.ax.plot(
+                self.fpr[0], self.tpr[0],
+                label='ROC for binary decision, AUC = {:0.2f}'.format(
+                        self.roc_auc[0]
+                )
+            )
+
+        # If per-class plotting is requested, plot ROC curves for each class
         if self.per_class:
```

Comment:
Just a quick check to make sure I'm following along - if `self._binary_decision is True`, then an exception will already have been raised if `per_class`, `micro`, or `macro` is true, which is why we can short-circuit the conditionals. I can't see the entirety of the function in the review; but if `self._binary_decision is True` can we return  from there, or is there work at the bottom of the function after the conditionals that needs to be done?

Reply:
There's a bit more logic that happens in the function. Here's the whole thing:

```
        if y_pred.ndim == 1:
            self._binary_decision = True

            # Raise an error if it's a binary decision and user has set micro,
            # macro, or per_class to True
            if self.micro or self.macro or self.per_class:
                raise ModelError(
                    "Micro, macro, and per-class scores are not defined for "
                    "binary classification for estimators with only "
                    "decision_function methods; set micro, macro, and "
                    "per-class params to False."
                )
        else:
            self._binary_decision = False
            # If it's not a binary decision, at least one of micro, macro, or
            # per_class must be True
            if not self.micro and not self.macro and not self.per_class:
                raise YellowbrickValueError(
                    "no curves will be drawn; specify micro, macro, or per_class"
                )
```